selfie-camera/src/main/java/org/odk/collect/selfiecamera/Camera.kt

```diff
@@ -0,0 +1,21 @@
+package org.odk.collect.selfiecamera
+
+import android.view.View
+import androidx.activity.ComponentActivity
+import org.odk.collect.androidshared.livedata.NonNullLiveData
+
+internal interface Camera {
+    fun initialize(activity: ComponentActivity, previewView: View)
+    fun takePicture(imagePath: String, onImageSaved: () -> Unit, onImageSaveError: () -> Unit)
+    fun startVideo(videoPath: String, onVideoSaved: () -> Unit, onVideoSaveError: () -> Unit)
+    fun stopVideo()
+
+    fun isRecording(): Boolean
```

Comment:
I'm wondering if this shouldn't be one of the states? `UNINITIALIZED`, `INITIALIZED`, `FAILED_TO_INITIALIZE` and `RECORDING`?

Reply:
Oh, interesting. I like that idea. It would mean a little more complexity in `CaptureSelfieActivity`, but keeps our `Camera` interface leaner. I'll give it a go.